src/computation/Computation.cc

```diff
@@ -0,0 +1,96 @@
+#include "Computation.h"
+
+#include <iostream>
+#include <sstream>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+#include "set_relation/set_relation.h"
+
+namespace iegenlib {
+
+/* Computation */
+
+void Computation::printInfo() {
+    std::ostringstream stmts;
+    std::ostringstream iterSpaces;
+    std::ostringstream execSchedules;
+    std::ostringstream dataReads;
+    std::ostringstream dataWrites;
+    std::ostringstream dataSpacesOutput;
+
+    for (const auto& it : stmtsInfoMap) {
```

Comment:
Make sure you are only using spaces.  No tabs allowed.  I can't tell in the viewer, so please just check in your editor.

Reply:
Yep, all soft tabs